### ISSUE TITLE
Unexpected file name from download_files()

### ISSUE THREAD

**Issue by bgrich** [CenterForOpenScience/osfr#45]
It looks like passing a file name to the path argument in `download_files()` causes the downloaded file to have a file name with the given path and the original OSF file name pasted together.

``` r
library(osfr)
download_files('5z2bh', path = 'public_test_file.csv')
#> Saving to filename: public_test_file.csvpublic_test_file.csv
#> Successfully downloaded file.
#> [1] "public_test_file.csvpublic_test_file.csv"
```

Reading through the documentation on `download_files()` I was unsure what the expected case should be. With the current function, if path is unspecified then the file is saved using the OSF file name in the current working directory (and not a temp directory as stated in the documentation).  Should the path argument only include the folders to save the file to?

I would assume there are three typical use cases here:

* The user doesn't specify the path and the file is saved to the working directory with the OSF file name. 
* The user specifies a path like "folder/test_file.md" and the file is saved to the specified folder and file name.
* The user specifies a path like "folder/" and the file is saved to the specified folder with the OSF file name.

I am happy to put together a pull request for this, but I wanted to get some guidance as to the intended or expected response. Thanks!

**Comment by chartgerink** [comment]
The premise for these things is that I messed up and that the docs are incongruent with what is programmed which is incongruent with what I intended it to do :rofl: 

I agree with your cases and would add
* The user specifies path as `test_file.md`, so save file in working directory under that name.

Feel free to adjust the docs! I will be making an `mvp` branch soon to structure some work using your and Tim's roadmap :)

**Comment by bgrich** [comment]
Thanks Chris! Working on this now. Thanks for the input.

**Comment by bgrich** [comment]
Fixed in PR #52.
